app/views/reports/regions/show.html.erb

```diff
@@ -324,4 +324,16 @@
       countryAbbreviation: '<%= CountryConfig.current[:abbreviation] %>'
     }).initialize();
   });
+  // Overdue toggle plage refresh scroll
+  window.addEventListener('DOMContentLoaded', (event) => {
+    const overdueToggleElement = document.getElementById('overdue-section');
```

Comment:
[non-blocking] Can we add `addEventListeners` to 'ltfu toggle' as well to preserve the scroll position?

Reply:
See more recent comment